code/obj/item/organs/tail.dm

```diff
@@ -0,0 +1,232 @@
+				user, "<span class='alert'>You [fluff] the coccygeal coruna of [src] onto the apex of [H == user ? "your" : "[H]'s"] sacrum![prob(1) ? " The tailbone wiggles happily." : ""]</span>",\
+				H, "<span class='alert'>[H == user ? "You" : "<b>[user]</b>"] [fluff][H == user && fluff == "press" ? "es" : "s"] the coccygeal coruna of [src] onto the apex of your sacrum![prob(1) ? " Your tailbone wiggles happily." : ""]</span>")
+			else	// Any other tail
+			//	H.tri_message("<span class='alert'><b>[user]</b> [fluff][fluff == "press" ? "es" : "s"] [src] onto the apex of [H == user ? "[his_or_her(H)]" : "[H]'s"] sacrum!</span>",\
+			//	user, "<span class='alert'>You [fluff] [src] onto the apex of [H == user ? "your" : "[H]'s"] sacrum!</span>",\
+			//	H, "<span class='alert'>[H == user ? "You" : "<b>[user]</b>"] [fluff][H == user && fluff == "press" ? "es" : "s"] [src] onto the apex of your sacrum!</span>")
+				boutput(user, "That [src] is way too meaty to fit there!")	// I'll make it work it later
+
+		else if (!H.organHolder.tail && H.organHolder.chest.op_stage >= 11.0)
+			attachment_successful = 1
+
+			var/fluff = pick("insert", "shove", "place", "drop", "smoosh", "squish")
+
+			H.tri_message("<span class='alert'><b>[user]</b> [fluff][fluff == "smoosh" || fluff == "squish" ? "es" : "s"] [src] up against [H == user ? "[his_or_her(H)]" : "[H]'s"] sacrum!</span>",\
+			user, "<span class='alert'>You [fluff] [src] up against [user == H ? "your" : "[H]'s"] sacrum!</span>",\
+			H, "<span class='alert'>[H == user ? "You" : "<b>[user]</b>"] [fluff][fluff == "smoosh" || fluff == "squish" ? "es" : "s"] [src] up against your sacrum!</span>")
+
+		if (attachment_successful)
+			if (user.find_in_hand(src))
+				user.u_equip(src)
+			H.organHolder.receive_organ(src, "tail", 3.0)
+			if (boned)
+				H.organHolder.tail.op_stage = 0.0
+			else
+				H.organHolder.tail.op_stage = 11.0
+			H.update_body()
+			H.bioHolder.RemoveEffect(src.failure_ability)
+			return 1
+
+		return 0
+
+	on_life(var/mult = 1)
+		if (!..())
+			return 0
+		if (src.get_damage() >= FAIL_DAMAGE && src.donor.mutantrace) // Humans dont need tails to not be clumsy idiots
+			donor.bioHolder.AddEffect(src.failure_ability, 0, 0, 0, 1)
+		return 1
+
+	on_removal()
+		if (src.failure_ability && src.donor.bioHolder && src.donor.mutantrace)
+			src.donor.bioHolder.AddEffect(src.failure_ability, 0, 0, 0, 1)
+
+	on_broken(var/mult = 1)
+		if(prob(2) && src.donor.mutantrace)
+			src.donor.change_misstep_chance(10)
+			src.donor.bioHolder.AddEffect(failure_ability)
+
+/* Turns out a human tailbone'll work just fine?
+/obj/item/organ/tail/human	// some dummy tail that doesnt exist cus not everyone has a tail
+	name = "human tail"
+	desc = "Humans don't have tails... do they? They don't and you shouldn't be seeing this."
+	organ_name = "tail"
+
+	on_removal()
+		qdel(src)	// Humans dont have tails!
+*/
```

Comment:
?

Reply:
Forgot to remove that. Previously, human tails and bone tails were two different things, where if a human mob didn't have a mutant race with visible tail and wasn't a skeleton, they'd get some weird dummy placeholder tail organ that'd delete itself if it ever got removed. It didn't work right, and I found out that just giving them the skeleton's tail would work just fine.

I'll remove that.